sap/cli/atc.py

```diff
@@ -65,6 +65,38 @@ def print_worklist_as_html_to_stream(run_results, stream, error_level=99):
     return 0 if ret < 1 else 1
 
 
+def print_worklist_as_checkstyle_xml_to_stream(run_results, stream, error_level=99):
+    """Print results as checkstyle xml to stream"""
+
+    CHECKSTYLE_VERSION = '8.36'
+    ERROR = 'error'
+    WARNING = 'warning'
+    INFO = 'info'
+    SEVERITY_MAPPING = {
```

Comment:
Can you please make SEVERITY_MAPPING configurable via command line parameters and also via Environment variable?

Reply:
How can I pass mapping as command line parameters and as env variable? Do you have examples?
Another question is as we decided earlier we should have this static mapping, but it could be overridden by proposed options? and priority is
1) command-line argument
2) env var
3) static mapping
Default value if the key isn't provided will be "info" or maybe "error"?

Reply:
ACK, your proposal looks reasonable. Go ahead and implement it.